tests/models/mistral/test_modeling_flax_mistral.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+import numpy as np
+
+from transformers import MistralConfig, is_flax_available, is_tokenizers_available
+from transformers.testing_utils import require_flax, slow
+
+from ...generation.test_flax_utils import FlaxGenerationTesterMixin
+from ...test_modeling_flax_common import FlaxModelTesterMixin, ids_tensor
+
+
+if is_flax_available():
+    import jax.numpy as jnp
+
+    from transformers.models.mistral.modeling_flax_mistral import (
+        FlaxMistralForCausalLM,
+        FlaxMistralForSequenceClassification,
+        FlaxMistralModel,
+    )
+
+
+if is_tokenizers_available():
+    from transformers import LlamaTokenizerFast
+
+
+class FlaxMistralModelTester:
```

Comment:
If we remove the `FlaxMistralForSequenceClassification` class we'll be able to copy this entire class `FlaxMistralModelTester` directly from `FlaxLlamaModelTester` right?

Reply:
The whole tester can't be copied, since the configurations are different. Mainly the parameter `num_key_value_heads` would go to the default valiew, meaning that allmost all tests would fail

Reply:
Great - thanks for the clarification @kiansierra!